lib/pg/execute.js

```diff
@@ -103,8 +103,7 @@ async function executePlainSQL(dbc, rawSql, rawValues) {
     values: rawValues
   })
 
-  DEBUG && DEBUG('sql > ', sql)
-  DEBUG && values && values.length > 0 && DEBUG('values > ', values)
```

Comment:
not sure whether we need to take `values` into account...did you check the runtime output for that?